microsoft-edge/extensions-chromium/publish/transfer-ownership.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Transferring ownership of an extension
+description: How to transfer ownership of a Microsoft Edge Extension.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.service: microsoft-edge
+ms.subservice: extensions
+ms.date: 02/05/2025
+---
+# Transferring ownership of an extension
+
+As an extension developer, you can seamlessly transfer ownership of your Microsoft Edge extension (add-on) to another party.  Transferring ownership of an extension can be essential for various reasons, such as organizational changes, project handovers, or streamlining management processes.  Your Microsoft Edge extension can continue to thrive under new ownership without any disruptions.
+
+Transferring ownership of an extension is sometimes called _reparenting_ an extension.
+
+
+<!-- ---------------------------------------------------------------------- -->
+## Initiating the ownership transfer request
+
+The current owner of the extension must send an email, as follows:
+
+1. Start composing an email to [ext_dev_support@microsoft.com](mailto:ext_dev_support@microsoft.com).
+ 
+1. Make the **Subject** of the email:
+
+   [Reparenting Request]: \<crx ID of the extension to be transferred\><!-- todo: define or expand crx -->
+
+1. In the email body, include the following information:
+   * The reason for transferring ownership.
+   * The name of the product.<!-- todo: of the extension? -->
```

Comment:
name of the "product", or "extension"?

Reply:
Outdated, closing.  [fresh comment](https://github.com/MicrosoftDocs/edge-developer/pull/3367/files#r1945168602)